rplugin/python3/deoplete/sources/deoplete_jedi/server.py

```diff
@@ -524,12 +524,17 @@ def completions(self, *args):
         restart the server.
         """
         if self._count > self.max_completion_count:
-            self._count = 0
             self.restart()
 
         self._count += 1
-        stream_write(self._server.stdin, args)
-        return stream_read(self._server.stdout)
+        try:
+            stream_write(self._server.stdin, args)
+            return stream_read(self._server.stdout)
+        except StreamError as exc:
+            log.error('Caught %s during handling completions(%s), '
+                      ' restarting server', exc, args)
```

Comment:
> will appear as a message in Neovim's console

I could imagine this being useful to be notified about problems, but it should not happen (too) often of course.  Maybe use some kind of limit here?
We should also ensure that the message is informative, but not too exhausting.